### DISCUSSION TITLE
Un peu de magie dans ce monde de brut ( magie utiliser pour faire des truc de brut bien evidament)

### DISCUSSION THREAD

**Discussion by Archimageifrit** [Crownicles/Crownicles#1110] [Nouvelles idées]
Autre idée mais on pourrai rajouter un peu de magie dans ce monde de brut . Avec un moyen quelconque de loot des item rare : les parchemins ! Qui aurons des effet pour certain quotidien avec le !daily ou en combat avec une attaque en plus ( avoué pouvoir lancé des boule de feu en combats sa donne en vie ! ) ou d'autres parchemin plus utilitaires comme certains évitant le gel ou la maladie !

**Comment by ghost** [comment]
je crois que draft est contre la magie toussa toussa

**Reply by Archimageifrit** [reply]
Tu veux dire comme mécanique de jeux ou dans le lors ? Parce que si c'est pour le lors comment marche les livre d'incantation , et puis l'alchimie peut être considéré comme une forme de magie ( je parle des potion )

**Comment by symsym-1629** [comment]
De mémoire ds le code du rédacteur c'est **aucune** fantaisie donc je suis désolé mais y'a aucune chance que cette suggestion passe

**Reply by Archimageifrit** [reply]
Ok Peut être si vous me le  dite je suppose que ça doit être vrais mais du coup les _livre d'incantation _ c'est quoi ?

**Reply by leuhvraiminoryo** [reply]
les items ne sont pas cannon, c'est pour ça que certains sont plutôt loufoques, et que tu peux obtenir une bombe atomique en la forgeant sous les instructions du sympathique forgeron. Ton idée est pas mal du tout, et reste sous la forme d'items avec ta proposition des parchemins (je verrais vraiment bien ça en nouvelle catégorie d'items, et si ça passe pas auprès de draft, on pourrait au moins récupérer certaines des idées, comme la protection contre un debuff, pour les rajouter en effets de potion), donc je trouve personnellement que ça passe (en "objets magiques" on a la catégorie des potions, ça gênerait pas d'avoir aussi des parchemins quoi), mais je suis pas sûr que draft aura le même avis. 

ps : désolé mon message est un peu brouillon, j'ai toujours du mal à bien organiser ma pensée, et puis là j'écris ça sur tel à mon réveil 🙂
